.circleci/config.yml

```diff
@@ -9,9 +9,9 @@
 
 ## IMPORTANT
 # If you change the `default_docker_image` version, also change the `cache_key` version
-var_1: &default_docker_image circleci/node:10.12
-var_2: &browsers_docker_image circleci/node:10.12-browsers
-var_3: &cache_key node-0.12-{{ checksum "yarn.lock" }}-{{ checksum "examples/program/yarn.lock" }}
+var_1: &default_docker_image circleci/node:10.14
```

Comment:
did we check that the 0.27 release notes covered this? should comment there as good citizens if not

Reply:
It's probably the Ubuntu 14 deprecation that's in the release notes alerayd